src/vma/util/hash_map.inl

```diff
@@ -142,10 +142,12 @@ inline int hash_map<K, V>::calc_hash(const K &key) {
 	pval = (uint8_t*)&key;
 	// start toggle from 1, as the keys are usually succeders, and gone through htons 
 	for (i = 0, j = 1; i < sizeof(K); ++i, j ^= 1) {
+		/* cppcheck-suppress objectIndex */
 		csum8[j] ^= *pval;
 		++pval;
 	}
 	// to 12 bit
+	/* cppcheck-suppress objectIndex */
```

Comment:
Do we know why cppcheck complains about it?
Can it be solved if we do reinterpret_cast<>() instead of c cast?

Reply:
I did not find a way